js/package.json

```diff
@@ -1,29 +1,140 @@
 {
-  "name": "arrow",
-  "version": "0.0.0",
-  "description": "",
-  "main": "lib/arrow.js",
+  "name": "apache-arrow",
+  "version": "0.1.2",
+  "main": "./targets/es5/cjs/Arrow.js",
+  "module": "./targets/es5/esm/Arrow.js",
+  "browser": "./targets/es5/umd/Arrow.js",
+  "jsnext:main": "./targets/es2015/esm/Arrow.js",
+  "esnext:main": "./targets/esnext/esm/Arrow.js",
+  "typings": "./typings/Arrow.d.ts",
+  "description": "Apache's columnar in-memory data format for big data analytics",
```

Comment:
How about "JavaScript implementation for Apache Arrow columnar in-memory format"

Reply:
@wesm good lookin out, how does, "Apache Arrow columnar in-memory format" sound? npm users already assume modules are JS

Reply:
SGTM